release/util/sanity_check_windows.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# This script automatically download ray and run the sanity check for Windows
+# in various Python version. This script requires conda command to exist.
+
+unset RAY_ADDRESS
+export RAY_HASH=$RAY_HASH
+export RAY_VERSION=$RAY_VERSION
+
+if [[ -z "$RAY_HASH" ]]; then
+    echo "RAY_HASH env var should be provided"
+    exit 1
+fi
+
+if [[ -z "$RAY_VERSION" ]]; then
+    echo "RAY_VERSION env var should be provided"
+    exit 1
+fi
+
+if ! [ -x "$(command -v conda)" ]; then
+    echo "conda doesn't exist. Please download conda for this machine"
+    exit 1
+else
+    echo "conda exists"
+fi
+
+echo "Start downloading Ray version ${RAY_VERSION} of commit ${RAY_HASH}"
+
+pip install --upgrade pip
+
+# This is required to use conda activate
+source "$(conda info --base)/etc/profile.d/conda.sh"
+
+PYTHON_VERSIONS=( "3.9" "3.10" "3.11" "3.12" )
+
+for PYTHON_VERSION in "${PYTHON_VERSIONS[@]}"
+do
+    env_name="${RAY_VERSION}-${PYTHON_VERSION}-env"
```

Comment:
use all caps for variables: `ENV_NAME` ?

Reply:
done